charts/hlf-k8s/Chart.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 name: hlf-k8s
 home: https://substra.org/
-version: 1.3.2
+version: 1.3.3
```

Comment:
Could you bump to 1.4? This introduce a BC break that I'm not very confortable to put in a minor version.

Reply:
Done